var/spack/repos/builtin/packages/py-backports-shutil-get-terminal-size/package.py

```diff
@@ -36,5 +36,5 @@ class PyBackportsShutilGetTerminalSize(PythonPackage):
 
     # newer setuptools version mess with "namespace" packages in an
     # incompatible way cf. https://github.com/pypa/setuptools/issues/900
```

Comment:
Is this comment no longer valid?

Reply:
I removed this on our internal spack some time ago and so far we didn't see any problems. But we are also only using our installations as views, where I'm not sure whether this would show up either way. I'd be fine with removing and seeing if anything breaks.

Reply:
I find that requiring an older version of setuputils causes a lot more problems for me than the namespaces issue. Any time I try to create a Python with a bunch of packages, the old setuputils requirement trips me up and I imagine trips up any Spack user. I don't know how many Spack users use Spack's module environment (and thus would be affected by the namespace issue), but I think that would affect less people. I believe one can also work around the namespace issue by activating the extension and then running through the python build.

Reply:
Are there any other packages that include this hack? We should also remove this comment if we are no longer requiring an old version of setuptools.

Reply:
My search indicates that this is the only package that depends_on a maximum version of setuptools. For legacy, I updated the comment to mention the previous "fix". That should help for anyone who this PR may break.

Reply:
The other thing that we might want to keep in mind is, that this is only a problem at all because we don't concretise build dependencies separately. So once this is fixed we wouldn't have to care about this at all anymore. So it might be an option to comment it out for now.